changelogs/current.yaml

```diff
@@ -124,6 +124,10 @@ minor_behavior_changes:
   change: |
     Sanitize SNI for potential log injection. The invalid character will be replaced by ``_`` with an ``invalid:`` marker. If runtime
     flag ``envoy.reloadable_features.sanitize_sni_in_access_log`` is set to ``false``, the sanitize behavior is disabled.
+- area: formatter
```

Comment:
Should this be runtime guarded?

Reply:
I am not sure. Because, this behavior change means:

 For a NaN value, the output of the command will be changed from `google.protobuf.Value cannot encode double values for nan, because it would be parsed as a string` to `null`.
For a Infinity value, the output of the command will be changed from `google.protobuf.Value cannot encode double values for infinity, because it would be parsed as a string` to `Inf`.

I doubt no one will depend on original behavior. So, not sure if it deserve a runtime flag.

Defer this to you to determine should we need to add a runtime flag for this. Anyway decision is fine to me. :)